Support/lib/executable.rb

```diff
@@ -94,19 +94,12 @@ def find(name, env_var = nil)
     # Return appropriate prefix for running commands via RVM if RVM is installed
     # and the current directory contains an RVM project file.
     def determine_rvm_prefix
-      rvm = "#{ENV['HOME']}/.rvm/bin/rvm"
-      %W(#{bundle_support_path}/bin/rvm_wrapper) if File.exist?(rvm)
+      rvm = "#{ENV['HOME']}/.rvm/bin/rvm-auto-ruby"
+      %W[#{ENV['HOME']}/.rvm/bin/rvm-auto-ruby] if File.exist?(rvm)
```

Comment:
I’m personally not a user of RVM, so please help me out: What is the issue you’re fixing here?

Some background: The `rvm_wrapper` shell script was added in https://github.com/textmate/ruby.tmbundle/pull/104/commits/149498e129c3a29d7ba3776eba208524921e1402 after a looooooong discussion (see https://github.com/textmate/ruby.tmbundle/pull/104#issuecomment-258960399 and the comments before and after). Back then we came to the conclusion that this script is necessary, for the reasons stated in the comments. But as said, I don’t know RVM well, so it’s perfectly possible that there is an easier solution.